FAQ.md

```diff
@@ -180,9 +178,8 @@ It is very easy to deloy Dragonfly in Kubernetes with [Helm](https://github.com/
 
 ## Can an image from a third-party registry be pulled via Dragonfly
 
-We **CANNOT** pull an image from a third-party registry via Dragonfly. Images from third-party registry means that the name of image has no registry address, for example, image `a.b.com/admin/mysql:5.6` is a third-party image, and it is from registry `a.b.com`. To the opposite, `admin/mysql:5.6` is not from third-party registry, because name of the image does not contain a registry address.
-
-Because administrator needs to configure `--registry-mirrors=["http://127.0.0.1:65001"]` for container engine to proxy part of image pulling requests to `dfdaemon` which listens on 65001 locally, images not from a third-part registry will use this registry mirror. However, when user pulls image `a.b.com/admin/mysql:5.6` which contains a third-party registry address, the container engine will directly access the third-party registry to pull the image, ignoring the configured registry mirror. Then It has nothing to do with the `dfdaemon` within Dragonfly.
+Yes, this can be achieved by [using dfdaemon as docker daemon's HTTP proxy](./docs/user_guide/proxy.md#download-images).
+**However if the registry has HTTPS enabled, currently images will be pulled without dfget.**
```

Comment:
The title of this question is `Can an image from a third-party registry be pulled via Dragonfly`, so it may be paradoxical to put this sentence here. Could you please keep it as it is, and I will update it later. THX.

Reply:
With `HTTP_PROXY` set to the transparent proxy, third party images can be pulled via dragonfly, though not necessarily with dfget. But I agree that we need better instructions for how to use dragonfly with third party registries, and I'll just revert it. It will be much easier if we solved the https hijacking problem.